src/pal/src/synchmgr/synchmanager.cpp

```diff
@@ -1835,16 +1878,36 @@ namespace CorUnix
             palErr = pSynchManager->ReadCmdFromProcessPipe(iPollTimeout, 
                                                            &swcCmd,
                                                            &shridMarshaledData,
-                                                           &dwData);            
+                                                           &dwData);
             if (NO_ERROR != palErr)
             {
                 ERROR("Received error %x from ReadCmdFromProcessPipe()\n",
-                      palErr);                
+                      palErr);
                 continue;
             }
             switch (swcCmd)
             {
-                case SynchWorkerCmdNop:                    
+                case SynchWorkerCmdTerminationRequest:
+                    // This worker thread is being asked to initiate process termination
+
+                    HANDLE hShutdownThread;
```

Comment:
This confused me a bit - it may be nice to call this `hThread` or `hTerminationRequestHandlingThread` to be consistent with the name of everything else.